simulation-ros/src/turtlebot2i/turtlebot2i_description/v-rep_model/turtlebot2i.lua

```diff
@@ -36,6 +64,12 @@ if (sim_call_type==sim.childscriptcall_initialization) then
     linVel = 0
     rotVel = 0
     motor_power = 1 --Enable motors by default
+------------------------Add by Enyu Cao----------------------
```

Comment:
'git blame' is a thing, BTW. no need to add "added by Enyu":

 
<img width="1366" alt="image" src="https://user-images.githubusercontent.com/64734/45387143-7d722080-b615-11e8-9c35-062509308e29.png">